### PR TITLE
Excersise 6

### PR DESCRIPTION
**Pull request by eherreraicalia**
The way I solved this problem

### CODE REVIEW

File: lalo/exercises/e06/StringCalc.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    def get_delimiters_brackets(self):
+        brackets = self.find_between('//', '\n')
+        return brackets
+
+    def get_delimiters(self):
+        pos_list = []
+        for pos in xrange(len(self.brackets)):
+            if self.brackets[pos] == '[' or self.brackets[pos] == ']':
+                pos_list.append(pos)
+        a = iter(pos_list)
```
Comment (kurenn) [inline lalo/exercises/e06/StringCalc.py]: `a` and `b` are not good variable names, try renaming them by giving the more meaningful names

File: lalo/exercises/e06/StringCalc.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            return self.string[start:end]
+        except ValueError:
+            return ""
+
+    def get_delimiters_brackets(self):
+        brackets = self.find_between('//', '\n')
+        return brackets
+
+    def get_delimiters(self):
+        pos_list = []
```
Comment (kurenn) [inline lalo/exercises/e06/StringCalc.py]: What does `pos_list` stands for?

File: lalo/exercises/e06/StringCalc.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                pos_list.append(pos)
+        a = iter(pos_list)
+        b = izip(a, a)
+        delimeters = []
+        for x, y in b:
+            delimeters.append(self.brackets[x+1:y])
+        return delimeters
+
+    def get_numbers(self):
+        sliced = self.string[len(self.brackets)+3:]
```
Comment (kurenn) [inline lalo/exercises/e06/StringCalc.py]: `sliced` is not a very good variable name

File: lalo/exercises/e06/StringCalc.py (hunk 1 above)
Comment (eherreraicalia) [reply lalo/exercises/e06/StringCalc.py]: yeah bad habbit

File: lalo/exercises/e06/StringCalc.py (hunk 2 above)
Comment (eherreraicalia) [reply lalo/exercises/e06/StringCalc.py]: it's a list containing the positions, where i found '[' or ']', I'll rename the variable

**Pull request merged**